crates/languages/src/python.rs

```diff
@@ -770,6 +770,12 @@ impl PyLspAdapter {
     }
 }
 
+#[cfg(not(target_os = "windows"))]
+const BINARY_DIR: &str = "bin";
```

Comment:
Nit: could we define `BINARY_DIR` just once and dispatch it's value based on `target_os` value like so?
```suggestion
#[cfg(not(target_os = "windows"))]
const BINARY_DIR: &str = if cfg!(target_os = "windows") {
  "Scripts"
} else {
  "bin"
};
```
```